docs/_static/spec/server.yaml

```diff
@@ -0,0 +1,1517 @@
+      description: >-
+        Runs the conversations tracker through the model's
+        policies to predict the scores of all actions present
+        in the model's domain. Actions are returned in the
+        'scores' array, sorted on their 'score' values.
+        The state of the tracker is not modified.
+      parameters:
+      - $ref: '#/components/parameters/conversation_id'
+      responses:
+        200:
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PredictResult'
+              example:
+                PredictResult:
+                  $ref: '#/components/examples/PredictResult'
+        401:
+          $ref: '#/components/responses/401NotAuthenticated'
+        403:
+          $ref: '#/components/responses/403NotAuthorized'
+        500:
+          $ref: '#/components/responses/500ServerError'
+
+
+  /conversations/{conversation_id}/messages:
+    post:
+      security:
+      - TokenAuth: []
+      - JWT: []
+      tags:
+      - Tracker
+      summary: Add a message to a tracker
+      description: >-
+        Adds a message to a tracker. This doesn't trigger
+        the prediction loop. It will log the message
+        on the tracker and return, no actions will be
+        predicted or run. This is often used together with the
+        predict endpoint.
+      parameters:
+      - $ref: '#/components/parameters/conversation_id'
+      - $ref: '#/components/parameters/include_events'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                text:
+                  type: string
+                  description: >-
+                    Message text
+                sender:
+                  type: string
+                  description: >-
+                    Origin of the message - who sent it
+                  enum:
+                    - user
+                parse_data:
+                  $ref: '#/components/schemas/ParseData'
+              required: ["message", "sender"]
+            example:
+              sender: user
+              message: hello
+              parse_data:
+                text: hello
+                entities: []
+                intent:
+                  confidence: 0.57
+                  name: greet
+                intent_ranking:
+                - confidence: 0.57
+                  name: greet
+                - confidence: 0.33
+                  name: goodbye
+      responses:
+        200:
+          $ref: '#/components/responses/200Tracker'
+        400:
+          $ref: '#/components/responses/400BadRequest'
+        401:
+          $ref: '#/components/responses/401NotAuthenticated'
+        403:
+          $ref: '#/components/responses/403NotAuthorized'
+        500:
+          $ref: '#/components/responses/500ServerError'
+
+  /models/train:
```

Comment:
I am not sure this is the best naming for the route (and the following routes), as `models` suggests this is a collection and the following segment is the model name / id.

we might get around this by doing something like `models/current/train` or `model/train` (other suggestions?) - keeping the `models/` route for actual model CRUD

Reply:
Good point. I will rename it to `model/train`.